site/app/templates/grading/electronic/Details.twig

```diff
@@ -60,7 +60,9 @@
     </div>
     {% if grade_inquiry_only_button %}
         <script>
-            inquiry_update();
+            $(document).ready(function(){
+                inquiry_update();
+            })
```

Comment:
Would it be better to put the document.ready in the function itself instead?

Reply:
I put the inquiry_update in the ending script tag and removed the jquery altogether.